transcription/transcription.go

```diff
@@ -11,13 +11,15 @@ import (
 
 // SendEmail connects to an email server at host:port, switches to TLS,
 // authenticates on TLS connections using the username and password, and sends
-// an email from address from, to address to, with subject line subject with message body.
+// an email from address from, to address to, with subject line subject with
+// message body.
 func SendEmail(username string, password string, host string, port int, to []string, subject string, body string) error {
```

Comment:
Would it be "cleaner" if we made this fit into character limit by breaking up into two lines?